start

```diff
@@ -26,10 +26,10 @@ export PGPORT=5432
 : "${ENABLE_LOGS:=false}"
 : "${ENABLE_CORE:=false}"
 : "${ENABLE_HORIZON:=false}"
-: "${ENABLE_SOROBAN_RPC:=false}"
+: "${ENABLE_STELLAR_RPC:=false}"
```

Comment:
This env var should lose the stellar prefix, see above for the others like core and horizon.
```suggestion
: "${ENABLE_RPC:=false}"
```